routes/x/module.tsx

```diff
@@ -375,87 +385,43 @@ function ModuleView({
   );
 
   if (data.view === "info") {
-    return <InfoView version={version!} data={data.data} name={name} />;
-  } else if (data.view === "source") {
     return (
-      <SourceView
-        {...{
-          isStd,
-          name,
-          version,
-          path,
-          url,
-          data: data.data,
-          repositoryURL,
-        }}
+      <InfoView
+        version={version!}
+        data={data.data}
+        name={name}
       />
     );
-  } else {
+  }
+
+  if (data.view === "source") {
     return (
-      <DocView
+      <SourceView
         {...{
           isStd,
           name,
           version,
           path,
           url,
-          data: data.data as DocPageSymbol | DocPageModule | DocPageIndex,
+          data: data.data,
           repositoryURL,
         }}
       />
     );
   }
-}
-
-function Breadcrumbs({
```

Comment:
for what reason are the breadcrumbs removed? I think this is fairly important as else there is no way to navigate backwards; also, from recent comments, it seems people actively use the breadcrumbs, so i wouldnt be in favour of removing it

Reply:
you are tight, and i perfer to put it in the "nav" place instead of as a title, will update

Reply:
<img width="500" alt="Screen Shot 2022-09-01 at 11 43 08" src="https://user-images.githubusercontent.com/2883484/187827096-572b64c8-8773-4bbc-b1ce-73f73bf0bf11.png">
<img width="500" alt="Screen Shot 2022-09-01 at 11 43 19" src="https://user-images.githubusercontent.com/2883484/187827098-3faef4d3-60f4-4754-aa94-9c5a25ebd307.png">

Reply:
I like that, though the _Jump to..._ dramatically changes the payload that we deliver per page.  right now, we only deliver the nav based on the current folder, that would seem to imply we would need to deliver a whole module structure on every page render.  It is a non trivial amount of work as well as has performance implications.

Also, how would it look with something more deeply nested like: https://dotland-17mf6x94e9mg.deno-staging.dev/x/cliffy@v0.24.3/command/upgrade/provider/github.ts

Reply:
![image](https://user-images.githubusercontent.com/2883484/187850004-c7e62607-192d-4bb3-afbe-7055bc7b685c.png)